app/assets/images/payment_icons/creditkey.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-creditkey"><title id="pi-creditkey">Credit Key</title><path d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z" opacity=".07"/><path d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32" fill="#fff"/><path d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z" opacity=".07"/><path d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32" fill="#fff"/><path d="M19.3 11.7v-4h4c0 2.2-1.9 4-4 4zm0 4.4v-4c2.2 0 4 1.7 4 4h-4zm-.6-4.4h-3.8c0-2.2 1.7-4 3.8-4v4zm0 4.4c-2.2 0-4-1.7-4-4h4v4zM25 6.8l-6-3.6-6 3.6c-.9.5-1.4 1.4-1.4 2.4v5.5c0 1 .5 1.9 1.4 2.4l6 3.6 6-3.6c.9-.5 1.4-1.4 1.4-2.4V9.3c0-1.1-.5-1.9-1.4-2.5z" fill="#3d9ce5"/></svg>
```

Comment:
There's a duplicated border so the border color looks darker. We will need to remove it.

[video demonstration](https://screenshot.click/09-02-bzud6-obtta.mp4)

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-creditkey"><title id="pi-creditkey">Credit Key</title><path d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z" opacity=".07"/><path d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32" fill="#fff"/><path d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32" fill="#fff"/><path d="M19.3 11.7v-4h4c0 2.2-1.9 4-4 4zm0 4.4v-4c2.2 0 4 1.7 4 4h-4zm-.6-4.4h-3.8c0-2.2 1.7-4 3.8-4v4zm0 4.4c-2.2 0-4-1.7-4-4h4v4zM25 6.8l-6-3.6-6 3.6c-.9.5-1.4 1.4-1.4 2.4v5.5c0 1 .5 1.9 1.4 2.4l6 3.6 6-3.6c.9-.5 1.4-1.4 1.4-2.4V9.3c0-1.1-.5-1.9-1.4-2.5z" fill="#3d9ce5"/></svg>
```